public/docs/ts/latest/guide/security.jade

```diff
@@ -0,0 +1,248 @@
+  generated authentication token in a cookie, often with the name `XSRF-TOKEN`. Cookies can only
+  be read by the website on which they are set, so only your own application can read this token. On
+  each API request, the server then validates the client by checking that the token is sent back,
+  usually in an HTTP header called `X-XSRF-TOKEN`.
+
+  The Angular `http` client has built-in support for this technique. The default
+  `CookieXSRFStrategy` looks for a cookie called `XSRF-TOKEN` and sets an HTTP request header named
+  `X-XSRF-TOKEN` with the value of that cookie on every request. The server must set the
+  `XSRF-TOKEN` cookie, and validate the response header for each state modifying request.
+
+  XSRF tokens should be unique per user and session, have a large random value generated by a
+  cryptographically secure random number generator, and expire.
+
+  Angular applications can customize cookie and header names by binding their own
+  `CookieXSRFStrategy` value, or implement an entirely custom `XSRFStrategy` by providing a custom
+  binding for that type, by adding
+  `provide(XSRFStrategy, {useValue: new CookieXSRFStrategy('myCookieName', 'My-Header-Name')})` or
+  `provide(XSRFStrategy, {useClass: MyXSRFStrategy})` to your providers list.
+
+  Learn about Cross Site Request Forgery (XSRF) at the Open Web Application Security Project (OWASP)
+  [here](https://www.owasp.org/index.php/Cross-Site_Request_Forgery_%28CSRF%29) and
+  [here](https://www.owasp.org/index.php/CSRF_Prevention_Cheat_Sheet). This [Stanford University
+  paper](https://seclab.stanford.edu/websec/csrf/csrf.pdf) is a rich source of detail.
+
+h3#xssi Cross-site Script Inclusion (XSSI)
+:marked
+  Cross-site Script Inclusion, also known as JSON vulnerability, can allow an attacker's website to
+  read data from a JSON API. The attack works on older browser by overriding native JavaScript
+  object constructors, and then including an API URL using a `<script>` tag.
+
+  This attack is only successful if the returned JSON is executable as JavaScript. Servers can
+  prevent it by prefixing all JSON responses to make them non-executable, by convention using the
+  well-known string `")]}',\n"`.
+
+  Angular's `Http` library recognizes this convention and automatically strips the string
+  `")]}',\n"` from all responses before further parsing.
+
+  Learn more in the XSSI section of this [Google web security blog
+  post](https://security.googleblog.com/2011/05/website-security-for-webmasters.html)
+
+.l-main-section
+h2#code-review Auditing Angular Applications
+:marked
+  Angular applications should follow the same security principles as regular web applications, and
+  should be audited as such. Angular specific APIs that should be audited in a security review,
+  such as the [_bypassSecurityTrust_](#bypass-security-apis) APIs, are marked in the documentation
+  as security sensitive.
```

Comment:
Is there a way to list all APIs that are marked as security sensitive in the API documentation? It would be nice to include a link here if possible.